### PR TITLE
Update FAQ to clarify use of GPL compatible license

### PR DESCRIPTION
**Pull request by aurindam**
Adds clarification on how someone can use Slint under GPL with code that is licensed under a GPL compatible license.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
   - [Why are you creating a new markup language?](#why-are-you-creating-a-new-markup-language)
   - [Will there be API bindings to integrate with my favorite programming language?](#will-there-be-api-bindings-to-integrate-with-my-favorite-programming-language)
 - [Licensing](#licensing)
-  - [Can I license my code under an open-source license more permissive than GPL?](#can-i-license-my-code-under-an-open-source-license-more-permissive-than-gpl)
+  - [If I link my program with Slint GPLv3, does it mean that I have to GPL my program too?](#if-i-link-my-program-with-slint-gplv3-does-it-mean-that-i-have-to-gpl-my-program-too)
```
Comment (tronical) [inline FAQ.md:8]:
```suggestion
  - [If I link my program with Slint GPLv3, does it mean that I have to license my program under the GPLv3, too?](#if-i-link-my-program-with-slint-gplv3-does-it-mean-that-i-have-to-gpl-my-program-too)
```

File: FAQ.md [hunk 2]
```diff
@@ -58,17 +60,21 @@ types of programming languages.
...
+Refer to GPL FAQ [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL).
 
-- [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL)
-- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)
+### My MIT-licensed program links to Slint GPLv3. Can someone fork my program to build and distribute a proprietary program?
 
-In other words, you can still allow others to fork your code for proprietary software.
-The person doing that have the choice to either use Slint according to one of our proprietary options, or remove the dependency.
+Yes, provided the person distributing the proprietary program either links to 
+Slint proprietary license instead of Slint GPLv3 or removes the dependency to Slint.
```
Comment (tronical) [inline FAQ.md:72]:
```suggestion
Yes, provided the person distributing the proprietary program acquired a 
Slint proprietary license instead of using Slint under GPLv3, or removes the dependency to Slint altogether.
```
